server/go/pkg/server/server.go

```diff
@@ -0,0 +1,295 @@
+package server
+
+import (
+	"context"
+	"io"
+	"log"
+
+	awssession "github.com/aws/aws-sdk-go/aws/session"
+	"github.com/godaddy/asherah/go/appencryption"
+	"github.com/godaddy/asherah/go/appencryption/pkg/crypto/aead"
+	"github.com/godaddy/asherah/go/appencryption/pkg/kms"
+	"github.com/godaddy/asherah/go/appencryption/pkg/persistence"
+	"github.com/godaddy/asherah/go/securememory/memguard"
+
+	pb "github.com/godaddy/asherah/server/go/api"
```

Comment:
any particular reason why we've renamed this import to pb and not just using `api`?

Reply:
Convention. The `pb` alias just calls attention to the fact that these types are provided by the protobuf code generator.